webtool/frontend/src/components/solarfarm/solarfarm-form.tsx

```diff
@@ -31,6 +32,21 @@ export const SolarFarmForm: FunctionComponent<{
                     <label className="form-label" htmlFor="nominalPower_kW">Vermogen (kW)</label>
                     <input className="form-input" type="number" id="nominalPower_kW" name="nominalPower_kW" defaultValue={ initialData?.nominalPower_kW || 1000}/>
                 </div>
+                <div className="radix-grid">
+                    <label className="form-label">Orientatie</label>
```

Comment:
I would call this label "Opstelling" and the option labels "Oost-west" and "Zuid"